introduction_to_amazon_algorithms/image_classification_tensorflow/Amazon_TensorFlow_Image_Classification.ipynb

```diff
@@ -0,0 +1,836 @@
+{
```

Comment:
<p>I would rephrase it so that the link appears in context with the sentence instead of an awkward separate phrase i.e.  For this example, that means on an input image, predicting the class label from one of the 1000 classes of the ImageNet dataset.  -&gt; "class label from one of the 1000 classes of the ImageNet dataset" is the link to ImageNetLabels</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3579/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>done.</p>